### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by rekmarks**
The unit tests make requests to Infura. This package was using the deprecated (removed?) Infura v1 API. This PR adds the `rc` dependency such that developers can add their Infura API keys for tests. It also adds the creation of an `rc` file for CI.

### CODE REVIEW

**Comment by rekmarks** [comment]
Notice how tests are failing for #26 at the moment.

**Review by Gudahtt — COMMENTED** [review]
This is better than failing tests, but it's still not great to be hitting Infura in unit tests :grimacing:. What would it take to replace this with `ganache` instead? We'd probably need to deploy this method registry contract I'm guessing.

**Review by astarinmymind — APPROVED** [review]
Looks good to me!

File: test/index.js [hunk 1]
```diff
@@ -1,8 +1,13 @@
 const test = require('tape');
 const Eth = require('ethjs');
+
+const { INFURA_PROJECT_ID } = require('rc')('infura', {
+  INFURA_PROJECT_ID: null,
```
[the remark this answers is not in the document]
Comment (rekmarks) [reply test/index.js:5]: Re: hitting Infura in unit tests, we are likely to run these tests a very small amount for the time being, and we can go through the motions of setting up `ganache` if we decide to do more development on it. We've touched this package for brief periods of time roughly once a year going back to 2019.

[the remark this answers is not in the document]
Comment (rekmarks) [reply test/index.js]: We'll set up Ganache the next time we work with this library. For now, we're using `process.env` as the default for the Infura project ID.

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**